reactor-core/src/test/java/reactor/core/publisher/FluxCollectTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2011-2020 Pivotal Software Inc, All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2011-Present Pivotal Software Inc, All Rights Reserved.
```

Reply:
This should not be `Pivotal` but `VMware`